### ISSUE TITLE
Is it necessary to use a multi-step validation process?

### ISSUE THREAD

**Issue by Tempo0o** [facebook/prophet#2369]
I am new to timeseries forecasting and from what I understand grid searching for the best model based on a testing set may lead to overfitting, hence the requirement for a validation set. I am seeing a lot of example in timeseries forecasting in general and in Prophet in particular where the model is only tested on one testing set. Even with cross-validation, where it seemed to me like enough validation, I read that you usually still want an untouched final set to test on.
Is it really necessary to have a multi-step validation process with Prophet, and if not, why exactly?
